sdk/storage/inc/blobs/blob_responses.hpp

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include "protocol/blob_rest_client.hpp"
+
+#include <limits>
+#include <string>
+#include <utility>
+
+namespace Azure { namespace Storage { namespace Blobs {
+
+  struct BlobDownloadInfo
+  {
+    std::string ETag;
+    std::string LastModified;
+    int64_t ContentLength = 0;
+    BlobHttpHeaders HttpHeaders;
+    std::map<std::string, std::string> Metadata;
+    Blobs::BlobType BlobType = Blobs::BlobType::Unknown;
+    Azure::Core::Nullable<bool> ServerEncrypted;
+    Azure::Core::Nullable<std::string> EncryptionKeySHA256;
+  };
+
+  struct PageRange
+  {
+    int64_t Offset;
+    int64_t Length;
+  };
+
+  struct PageRangesInfo
+  {
+    std::string RequestId;
```

Comment:
Do we still need to have the fabulous 4 here?

Reply:
fixed